types/mailparser/index.d.ts

```diff
@@ -177,6 +187,10 @@ interface ParsedMail {
 	 * - `date` value is a Date object.
 	 */
 	headers: Headers;
+    /**
+	 * An array of raw header lines
+	 */
+	headerLines: HeaderLines;
```

Comment:
I know nothing about the underlying library, but it looks like `false` might also be a valid value?
https://github.com/nodemailer/mailparser/blob/9636083b806e60a953a640a4f833a700931b5618/lib/mail-parser.js#L137

Reply:
You're right. `Headers` doesn't allow for a `false` value either despite being initialized as false. Both should be fine though - this is an email parsing library - receiving an email with no headers is highly unlikely (from, to, subject, etc. are headers).